package-lock.json

```diff
@@ -4200,13 +4200,13 @@
         },
         "ansi-styles": {
           "version": "2.2.1",
-          "resolved": "https://registry.npmjs.org/ansi-styles/-/ansi-styles-2.2.1.tgz",
+          "resolved": "https://registry.npm.taobao.org/ansi-styles/download/ansi-styles-2.2.1.tgz",
```

Comment:
No, I mean, we don't want to have this file checked in at all for this PR (and also do not want to use registry.npm.taobao.org)

Reply:
oh, weird. I thought I removed this.